initiator.go

```diff
@@ -183,6 +183,9 @@ type IscsiOptions struct {
 
 	// Scheduler to configure for the blockdev
 	Scheduler string
+
+	// ScanTimeout is the total time to wait for devices to appear via uevents.
```

Comment:
technically we're just waiting for the block device to appear in the filesystem, not listening to uevents (which happens over netlink)